parcel_test.go

```diff
@@ -108,14 +170,20 @@ func TestGetByClient(t *testing.T) {
 	}
 
 	// get by client
-	storedParcels, err := // получите список посылок по идентификатору клиента, сохранённого в переменной client
+	// получите список посылок по идентификатору клиента, сохранённого в переменной client
+	storedParcels, err := store.GetByClient(client)
 	// убедитесь в отсутствии ошибки
+	require.NoError(t, err)
 	// убедитесь, что количество полученных посылок совпадает с количеством добавленных
+	require.Equal(t, len(parcels), len(storedParcels))
 
 	// check
 	for _, parcel := range storedParcels {
 		// в parcelMap лежат добавленные посылки, ключ - идентификатор посылки, значение - сама посылка
 		// убедитесь, что все посылки из storedParcels есть в parcelMap
+		p, err := parcelMap[parcel.Number]
```

Comment:
В целом все ок, кроме названия переменной. Обычно переменная `err` имеет тип `error`, а тут `bool`. Вот несколько вариантов получше
```suggestion
		p, ok := parcelMap[parcel.Number]
```
```suggestion
		p, exists := parcelMap[parcel.Number]
```